requirements.txt

```diff
@@ -15,4 +15,4 @@ kubernetes >= 24.2.0, < 30.0.0
 pytz >= 2021.1, < 2025
 readchar >= 4.0.0, < 5.0.0
 sqlalchemy[asyncio] >= 1.4.22, != 1.4.33, < 3.0.0
-typer >= 0.4.2
+typer >= 0.12.0
```

Comment:
👍  👎  ?
```suggestion
typer >= 0.12.0< 0.13.0
```

Reply:
I'd vote 👎 , upper bounds tend to be pretty annoying for downstream users. They're fine if we have a known incompatibility, but I wouldn't, in general, favor pre-emptive pinning.